hail/src/main/scala/is/hail/io/gen/ExportGen.scala

```diff
@@ -111,34 +110,31 @@ class GenAnnotationView(rowType: PStruct) extends View {
   private val rsidIdx = rsidField.index
   private val varidIdx = varidField.index
 
-  private var region: Region = _
   private var rsidOffset: Long = _
   private var varidOffset: Long = _
 
   private var cachedVarid: String = _
   private var cachedRsid: String = _
 
   def setRegion(region: Region, offset: Long) {
```

Comment:
All calls to setRegion(region: *) can be migrated to setRegion(rv) or even more clearly setRegion(rv.offset). I elected not to do this here, since it's a bit out of scope of a "remove region from PType methods PR", but left a FIXME on View. Your call what to do.

Reply:
do it in a followup, that sounds right.

Reply:
https://github.com/hail-is/hail/issues/7908